backend_new/test/unit/services/listing.service.spec.ts

```diff
@@ -836,177 +836,293 @@ describe('Testing listing service', () => {
             updatedAt: date,
-            id: 'unitType 3',
-            name: UnitTypeSort[3],
+            name: 'twoBdrm',
             numBedrooms: 3,
           },
-          minIncomeRange: {
-            max: '$8',
-            min: '$3',
-          },
-          occupancyRange: {
-            max: 8,
-            min: 3,
-          },
-          rentAsPercentIncomeRange: {
-            max: 3,
-            min: 3,
-          },
-          rentRange: {
-            max: '$8',
-            min: '$3',
-          },
-          totalAvailable: 2,
-          areaRange: {
-            max: 8,
-            min: 3,
-          },
-          floorRange: {
-            max: 8,
-            min: 3,
-          },
+          totalAvailable: 1,
         },
         {
+          areaRange: { min: 4, max: 4 },
+          minIncomeRange: { min: '$4', max: '$4' },
+          occupancyRange: { min: 4, max: 4 },
+          rentRange: { min: '$4', max: '$4' },
+          rentAsPercentIncomeRange: { min: 4, max: 4 },
+          floorRange: { min: 4, max: 4 },
           unitTypes: {
+            id: 'unitType 4',
             createdAt: date,
             updatedAt: date,
-            id: 'unitType 4',
-            name: UnitTypeSort[4],
+            name: 'threeBdrm',
             numBedrooms: 4,
           },
-          minIncomeRange: {
-            max: '$4',
-            min: '$4',
-          },
-          occupancyRange: {
-            max: 4,
-            min: 4,
-          },
-          rentAsPercentIncomeRange: {
-            max: 4,
-            min: 4,
-          },
-          rentRange: {
-            max: '$4',
-            min: '$4',
-          },
           totalAvailable: 1,
-          areaRange: {
-            max: 4,
-            min: 4,
+        },
+        {
+          areaRange: { min: 5, max: 5 },
+          minIncomeRange: { min: '$5', max: '$5' },
+          occupancyRange: { min: 5, max: 5 },
+          rentRange: { min: '$5', max: '$5' },
+          rentAsPercentIncomeRange: { min: 5, max: 5 },
+          floorRange: { min: 5, max: 5 },
+          unitTypes: {
+            id: 'unitType 5',
+            createdAt: date,
+            updatedAt: date,
+            name: 'fourBdrm',
+            numBedrooms: 5,
           },
-          floorRange: {
-            max: 4,
-            min: 4,
+          totalAvailable: 1,
+        },
+        {
+          areaRange: { min: 6, max: 6 },
+          minIncomeRange: { min: '$6', max: '$6' },
+          occupancyRange: { min: 6, max: 6 },
+          rentRange: { min: '$6', max: '$6' },
+          rentAsPercentIncomeRange: { min: 6, max: 6 },
+          floorRange: { min: 6, max: 6 },
+          unitTypes: {
+            id: 'unitType 6',
+            createdAt: date,
+            updatedAt: date,
+            name: 'fiveBdrm',
+            numBedrooms: 6,
           },
+          totalAvailable: 1,
         },
       ],
     });
+    // expect(res.items[0].unitsSummarized).toEqual({
```

Comment:
Is this going to be used later?

Reply:
That was just for reference. It has been deleted